BetterPasswordPlugin.inc.php

```diff
@@ -304,4 +377,27 @@ function addUserSettings($hookname, $args) {
 		);
 		return false;
 	}
+        /**
+	 * Callback to fill cache with data, if empty.
+	 * @param $cache FileCache
+	 * @param $password_hash string The leading two characters of the hash users password
```

Comment:
This is no longer the leading two characters, but the full hashed password, I think.